weather_sp/splitter_pipeline/file_name_utils.py

```diff
@@ -51,21 +48,21 @@ def get_output_file_base_name(filename: str,
             The output file is then created by replacing this part of the input name
             with the output pattern.
     """
-    file_ending = ''
-    split_name, ending = os.path.splitext(filename)
-    if ending in [*GRIB_FILE_ENDINGS, *NETCDF_FILE_ENDINGS]:
-        file_ending = ending
-        filename = split_name
+    filename, ending = os.path.splitext(filename)
```

Comment:
I'm pretty sure you suggested this in a previous change and I already argued against it :)
Just checking some example files I have for testing, there's a file '20210101_gfs.t00z.pgrb2.0p25.f012'. I'm fairly certain our gfs gribs follow this king of patter ending with '.pgrb2.0p25.f0xy' depending on the step for the forecast time. 'f012' should no be used as ending.

Reply:
Ah, now these lines make sense, thanks for the context, I'll add them back.